modules/objdetect/samples/python/detect_board_charuco.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
```

Comment:
`modules/objdetect/samples` is bad location. should be `samples/python`

Reply:
sample was moved